SPECS/build-essential/build-essential.spec

```diff
@@ -1,21 +1,25 @@
 Summary:        Metapackage to install all build tools
 Name:           build-essential
-Version:        0.1
-Release:        5%{?dist}
+Version:        3.0
```

Comment:
Thoughts on moving this to semantic versioning so we adhere to a standard versioning schema if one does not already exist?